### PR TITLE
Move Deployment Target to iOS 13.0.

### PR DESCRIPTION
**Pull request by brandonpage**

### CODE REVIEW

**Review by wmathurin — APPROVED** [review]

File: libs/MobileSync/MobileSync/Classes/Extensions/MobileSync.swift [hunk 1]
```diff
@@ -77,7 +77,7 @@ extension SyncManager {
     }
 }
 
-@available(iOS 13.0, watchOS 6.0, *)
+@available(watchOS 6.0, *)
```
Comment (bhariharan) [inline libs/MobileSync/MobileSync/Classes/Extensions/MobileSync.swift:80]: Can't we remove this `@available` entirely?

File: libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Extensions/RestClient.swift [hunk 2]
```diff
@@ -226,7 +226,7 @@ extension RestClient {
   
 }
 
-@available(iOS 13.0, watchOS 6.0, *)
+@available(watchOS 6.0, *)
```
Comment (bhariharan) [inline libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Extensions/RestClient.swift:229]: Same here - can't we remove this?

File: libs/SmartStore/SmartStore/Classes/Extensions/SmartStore.swift [hunk 3]
```diff
@@ -71,7 +71,7 @@ extension SmartStore {
     }
 }
 
-@available(iOS 13.0, watchOS 6.0, *)
+@available(watchOS 6.0, *)
```
Comment (bhariharan) [inline libs/SmartStore/SmartStore/Classes/Extensions/SmartStore.swift:74]: Remove this?

**Review by bhariharan — APPROVED** [review]
Could we hold off on merging this until next week when I get my auth changes in? I need to consume those changes back in `Communities` without bumping up the minimum version to `iOS 13`.

**Review by bhariharan — APPROVED** [review]

**Review by bhariharan — APPROVED** [review]

**Pull request merged**